config.go

```diff
@@ -17,6 +18,7 @@ func ImportConfig() Configuration {
 		Dbname:        "bibletagapi",
 		Dbaddress:     "localhost:28015",
 		TagPostTable:  "tags",
+		DBPAPIKey:		 nil
```

Comment:
This needs to be changed to `""` instead of `nil` because Go will barf on you if the types don't match.